webapp/src/components/pages/Home.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <div
+    class="mx-0 lg:mx-8 text-slate-600 dark:text-slate-300"
+  >
+
+  <!-- First screen -->
+    <div class="mb-20 lg:mb-0 lg:h-dvh flex flex-col justify-around pb-12">
+  <!-- Disco logo -->
+    <div class="flex flex-col justify-center items-center space-y-4">
+        <div class="">
+          <DiscoGIF v-if="themeStore.currentTheme == 'light'" class="pb-4"/>
+          <DiscoLogo v-else class="w-[640px] h-[264px]"/>
+        </div>
+        <span
+          class="text-xl sm:text-2xl md:text-3xl lg:text-4xl text-slate-600 dark:text-slate-300"
+        >
+          <span class="font-disco text-disco-cyan font-semibold">DIS</span
+          >tributed
+          <span class="font-disco text-disco-blue font-semibold">CO</span
+          >llaborative Learning
+        </span>
+      </div>
+      <div class="text-center md:text-left mt-24">
```

Comment:
while the rest of the page ~grow upon large screen, the header only does so when reaching medium screen, which is a bit weird

```suggestion
      <div class="text-center lg:text-left mt-24">
```